docs/user/gcp/limits.md

```diff
@@ -1,14 +1,86 @@
 # Handling quota on GCP
-All cloud providers have some concept of limits imposed to protect their services from misuse as well as protect customers from mistakes that can leads to excessive charges.  In GCP these will vary by region, project and account.  This is great news in terms of flexibility but can present an initial hurdle if the defaults in your project are too low.
+Quota limits vary by region, project and account. You can view the limits for your project here:
 
-## Baseline usage
-A vanilla IPI installation will result in 24 CPUs, 3 Static IPs and 768 GB of storage consumed.
+[GCP: IAM & Admin > Quotas][gcp-console-quota]
 
-## Common problems
-Be sure to consider cluster growth and consumption from other clusters if using a shared projectd.  The most likely areas of contention are CPU, Static IPs and Storage (SSD) quota.  Whenever an installation fails the installer CLI will return the relevant error message stating which quota was exceeded in a particular region.
+## Significant Resource Usage & Quotas
+These resources seem to be the most likely to encounter quota limit issues when installing a cluster on GCP. If you encounter a quota issue with a resource not on this list, please [let us know](https://github.com/openshift/installer/issues/new).
+
+### Storage
+A standard OpenShift installation utilizes 768 GB of storage (Persistent Disk SSD). Several regions have a default storage quota of 500 GB, which would cause a minimal installation to fail due to exceeding quotas. 
+
+Regions with a default limit of 500 GB are:
+* asia-east2
+* asia-northeast2
+* asia-south1
+* australia-southeast1
+* europe-north1
+* europe-west2
+* europe-west3
+* europe-west6
+* northamerica-northeast1
+* southamerica-east1
+* us-west2 
+
+If you must install to one of these regions and cannot install to a region with a higher quota, you will need to request a quota increase, as described in [the example below](#increasing-limits).
+
+### Firewall Rules
+An OpenShift cluster utilizes ~40 firewall rules. Some projects may have a quota as low as 200 firewall rules, which would limit the project to a maximum of five cluster installations. You can [check the firewall quota for your project here](https://console.cloud.google.com/iam-admin/quotas?service=compute.googleapis.com&metric=Firewall%20rules&_ga=2.10895548.-966099186.1563042928).
+
+### Compute
+A standard OpenShift installation uses 24 CPUs: each master and worker are allotted 4 CPUs. 
```

Comment:
"each master and worker are all" -> "each control-plane and compute machine is", see #1330.

Reply:
Oh, right. That was unintentional despite the fact that I object to calling masters "control plane" (kubelet on compute nodes is part of control plane), but that's what we've decided to go with so I'll change it. Thanks for catching that.